Unstructured-IO/pipeline-template
# chore: re-generated APIs after version bump

Also includes non-functional minor tweaks, fixes to ordering in `app.py`.

**Test Instructions**

Run `./check-generated-repo.sh` and rejoice when it completes successfully. Note that it is checking for output consistent with unstructured-api-tools==0.4.8 (which will get pip compiled, pip installed per the script as of writing).
